jellyfinclient/src/App.js

```diff
@@ -9,6 +9,7 @@ import configureStore from "./utilities/storage/store";
 import { Router, Switch, Route } from './utilities/routing/index';
 import { PersistGate } from 'redux-persist/integration/react';
 import Loading from './components/Loading';
+import HomeComponent from './components/HomeComponent';
```

Comment:
Shouldn't we at least try to be consistent with our import statements? When to use curlies and when not to?

Could  be boy scouted once (everywhere) and we should stick to some kind of convention afterwards. What do you think (@all)

Reply:
There is a difference between an import using curlies and non-curly ones.

Reply:
Yep, I like conventions too. But I have no experience with eslint so I will ignore this issue for now i guess 😆